### ISSUE TITLE
Error in Workflow (0.6.0-alpha.7)

### ISSUE THREAD

**Issue by ckoeberle** [idpaterson/alfred-wunderlist-workflow#112]
Hi Ian,

I'm not able to use the newest release (see screenshots)
![bildschirmfoto 2016-03-31 um 17 24 57](https://cloud.githubusercontent.com/assets/3305222/14181083/e6cd6ef2-f765-11e5-9783-d629217e0043.png)

![bildschirmfoto 2016-03-31 um 17 25 15](https://cloud.githubusercontent.com/assets/3305222/14181091/edd747a4-f765-11e5-91fa-92203c1a017a.png)

Commands like "wl-due" or the standard "wl" command work like expected.
I tried to completely clean everything (uninstall, _wl-reset, _wl-full-reset). Do you have any hints for me?

Best regards
Christoph

**Comment by idpaterson** [comment]
It looks like your lists may not have been cached for some reason. That may normally be the case for a short while after starting a fresh sync, but unless the sync fails outright it should still build out that lists.cpickle file. I wonder if I built with the wrong version of alfred-workflow where this was previously an issue, but I am unsure since it's not affecting me right now...

Kudos on finding the new reset commands. Could you please try a `_wl-reset` to clear the database, then run a sync in the terminal to see if there are any overt errors? Just change the path if you sync Alfred elsewhere:

```sh
python ~/Library/Application\ Support/Alfred\ 2/Alfred.alfredpreferences/workflows/user.workflow.*/alfred-wunderlist-workflow.py "pref sync" --commit
```

Then check whether there is a lists.cpickle file here:

```sh
ls ~/Library/Application\ Support/Alfred\ 2/Workflow\ Data/com.ipaterson.alfred.wunderlist/
```

**Comment by ckoeberle** [comment]
I get „Sync completed successfully“ but no lists.cpickle file …

The following files are in the directory:

   - .prefs.alfred-workflow
   - prefs.cpickle
   - settings.json
   - wunderlist.db

**Comment by idpaterson** [comment]
Does [the attached workflow](https://github.com/idpaterson/alfred-wunderlist-workflow/files/198042/Wunderlist.alfredworkflow.zip) behave any differently for you?

**Comment by ckoeberle** [comment]
Sorry, no changes, same error.

Here's what I did:

* completely remove the workflow
* delete directory com.ipaterson.alfred.wunderlist
* install the workflow
* log in
* run the command ```python ......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.*/alfred-wunderlist-workflow.py "pref sync" --commit```

__result:__
````
18:26:17 workflow.py:2172 ERROR    Instance matching query does not exist:
SQL: SELECT "t1"."id", "t1"."user_id", "t1"."revision" FROM "root" AS t1
PARAMS: []
Traceback (most recent call last):
  File "/Users/......./Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/workflow/workflow.py", line 2165, in run
    func(self)
  File "......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/alfred-wunderlist-workflow.py", line 11, in main
    route(wf.args)
  File "/Users/......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/wunderlist/handlers/route.py", line 72, in route
    handler.commit(command, modifier)
  File "/Users/......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/wunderlist/handlers/preferences.py", line 226, in commit
    sync()
  File "/Users/......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/wunderlist/sync.py", line 33, in sync
    root.Root.sync()
  File "/Users/......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/wunderlist/models/root.py", line 21, in sync
    instance = cls.get()
  File "/Users/......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/peewee.py", line 4505, in get
    return sq.get()
  File "/Users/......./Alfred/Alfred.alfredpreferences/workflows/user.workflow.F2185E63-FC47-4C4C-87DB-AC65780BAF42/peewee.py", line 2889, in get
    % self.sql())
RootDoesNotExist: Instance matching query does not exist:
SQL: SELECT "t1"."id", "t1"."user_id", "t1"."revision" FROM "root" AS t1
PARAMS: []
18:26:17 workflow.py:2190 DEBUG    Workflow finished in 0.099 seconds.
````

**Comment by idpaterson** [comment]
That error may be due to the initial sync which was already running in the background after logging in. Still no lists.cpickle file though?

**Comment by ckoeberle** [comment]
no, still no lists.cpickle file :/

**Comment by idpaterson** [comment]
Let's try [this one](https://github.com/idpaterson/alfred-wunderlist-workflow/files/198103/Wunderlist.alfredworkflow.zip)

**Comment by ckoeberle** [comment]
Now the python script says "success" again, but still no "lists.cpickle" file and the same errors ...

**Comment by idpaterson** [comment]
Hmm, I'm struggling to find something that may have changed in alpha.7 to cause this. Have you added or renamed any lists recently? I wonder if there could be an encoding issue... if you've made changes can you share the name of the list(s) that changed?

**Comment by ckoeberle** [comment]
I did not change my lists recently, but I have (had) a list with a german "Umlaut" in it.
The list name was "Urlaubsübergabe". I changed it to "Urlaubsuebergabe" and now everything is fine!

* wl-search works :+1: 
* wl_pref works :+1: 
* lists.cpickle file is available :+1: 

Thank you very much for your help! Fantastic support!
Maybe you'll find out how to fix the encoding problem, until then I can live without "äöüÄÖÜß" ;-)

Greets
Christoph
